google/resumable_media/_download.py

```diff
@@ -51,11 +52,12 @@ class DownloadBase(object):
     """
 
     def __init__(self, media_url, stream=None,
-                 start=None, end=None, headers=None):
+                 start=None, end=None, user_agent=None, headers=None):
```

Comment:
Is there any particular reason to put `user_agent` before `headers`?

Reply:
Nothing as such, i thought of having str argument(user_agent) before a dict(headers) looks good.